extension_templates/forecasting.py

```diff
@@ -238,19 +239,62 @@ def _update_predict_single(
 
     # todo: consider implementing this, optional
     # if not implementing, delete the method
-    def _compute_pred_int(self, alphas):
```

Comment:
important: we also need to remove the "old" prediction interval related arguments from `_predict`, and similarly from the docstring preamble at the top (`return_pred_int` etc)